cli/testdata/coder_server_--help.golden

```diff
@@ -426,6 +426,10 @@ OIDC OPTIONS:
       --oidc-icon-url url, $CODER_OIDC_ICON_URL
           URL pointing to the icon to use on the OpenID Connect login button.
 
+      --oidc-signups-disabled-text string, $CODER_OIDC_SIGNUPS_DISABLED_TEXT
+          The custom text to show on the error page informing about disabled
+          OIDC signups.
```

Comment:
This seems to support both markdown and HTML? Should we mention this or add some additional documentation on what is and is not allowed here?

Reply:
I disabled HTML tags on purpose, only Markdown content is rendered and every HTML is stripped. I can add a comment to `--help`.

Reply:
Done